src/assets/sass/protocol/components/_button.scss

```diff
@@ -4,36 +4,151 @@
 
 @import '../includes/lib';
 
+/* -------------------------------------------------------------------------- */
+// Primary Button
+
 .mzp-c-button {
     @include border-box;
     @include text-cta;
-    transition: background-color 100ms, border-color 100ms, color 100ms;
+    transition: all 200ms;
     background-color: $color-black;
     border-radius: 2px;
     border: 2px solid $color-black;
-    box-shadow: 0 4px 12px rgba(0, 0, 0, .1);
     color: $color-white;
     cursor: pointer;
     display: inline-block;
     font-weight: bold;
-    padding: $padding-md $padding-lg;
+    padding: calc(#{$padding-md} - 2px) calc(#{$padding-xl} - 2px); //2px extra padding removed to compensate for 2px border.
     text-decoration: none;
 
-    &:hover,
-    &:focus {
-        background-color: $color-white;
+    &:hover {
+        background-color: transparent;
         color: $color-black;
+        box-shadow: 4px 4px 0 0 $color-black;
+        transform: translate(-2px, -2px);
+    }
+
+    &:disabled {
```

Comment:
@alexgibson I included the disabled state because it was missing from our CSS file and having a button that is disabled but looks and functions like a non-disabled button is a really bad experience. 

But I think we should avoid disabled buttons whenever possible. A better way to handle things is to enable the button and give the user a notification when they can't continue because of missing information (this is how the sign up form works).

So maybe we shouldn't include an example. My hope is that if someone doesn't know its a possibility then they won't ask for it.